jc-1844 gs updates (#620)
redhat-developer/app-services-guides: docs/connectors/getting-started-connectors/README.adoc

Before you use {product-connectors}, you must complete the following prerequisites: 

> * Determine which {openshift} environment to use for your _{connectors} namespace_. The {connectors} namespace is where your {product-connectors} instances will be deployed.

Reviewer: Now that you have the description of "namespace" in this section, perhaps remove the text "The namespace is the deployment space that hosts your Connectors instances." from both the "Creating a Connectors instance for a data source" and "Creating a Connectors instance for a data sink" sections?

Reviewer: Resolving as Done :)

* Configure {product-long-kafka} for use with {product-connectors}.

*Determining which {openshift} environment to use for your {connectors} namespace*

You have three choices:

* *The hosted preview environment*

> ** The {connectors} instances are hosted on a multitenant {osd-name-short} cluster that is owned by Red Hat.

Reviewer: Perhaps rename the attribute from `osd-name-short` to `product-osd`?
You might want to create a `product-long-osd` attribute too, for the first occurrence?

Reviewer: My apologies, I steered you in the wrong direction with this suggestion. :(

As discussed on chat, we probably don't need two attributes for this product, especially now that we have confirmation that the short form "OSD" is not approved.

** You can create four {connectors} instances at a time.
** The preview environment applies 48-hour expiration windows, as described in https://access.redhat.com/documentation/en-us/openshift_connectors/1/guide/8190dc9e-249c-4207-bd69-096e5dd5bc64[Red Hat {openshift} {connectors} Preview guidelines^].

> * *Your own trial {osd-name-short} environment*

Reviewer: Resolving as Done -- thank you!

> ** You have access to your own {osd-name-short} trial environment.

Reviewer: Change "your own" to "an", for consistency with the body text for ROSA. Or perhaps change the ROSA entry to "your own", if that distinction is important. Or leave as is, if both are correctly different.

```suggestion
** You have access to an {osd-name-short} Trial environment.
```

** You can create an unlimited number of {connectors} instances.

> ** Your {osd-name-short} trial cluster expires after 60 days.

Reviewer: Resolving as Done :)

Before you use {product-connectors}, you must complete the following prerequisites: 

* Determine which {openshift} environment to use for your _{connectors} namespace_. The {connectors} namespace is where your {product-connectors} instances are deployed.

* Configure {product-long-kafka} for use with {product-connectors}.

*Determining which {openshift} environment to use for your {connectors} namespace*

You have three choices:

* *The hosted preview environment*

> ** The {connectors} instances are hosted on a multitenant {product-osd} cluster that is owned by Red Hat.

Reviewer: Ignore the above comment -- we should have only one osd attribute.

** You can create four {connectors} instances at a time.
** The preview environment applies 48-hour expiration windows, as described in https://access.redhat.com/documentation/en-us/openshift_connectors/1/guide/8190dc9e-249c-4207-bd69-096e5dd5bc64[Red Hat {openshift} {connectors} Preview guidelines^].

> * *Your own {product-osd-long} Trial environment*

Reviewer: Ignore the above comment -- we should have only one osd attribute.

> ** You have access to an {product-osd-long} Trial environment.

Reviewer: Ignore the above comment -- we should have only one osd attribute, and you've replaced the article with "your own" to avoid the need to change.

For this example, you create an instance of the Data Generator source connector. The Data Generator is provided for development and testing purposes. You specify the text for a message and how often to send the message.

.Prerequisites

> * If you want to use a dead letter queue (DLQ) to handle any messaging errors, create a Kafka topic to use for the DLQ.

Reviewer: Now that you've added this DLQ text, you might want to tweak the existing text that refers to the topic for the **Dead letter queue** error handling policy (in both the Source and Sink sections)?

> * If you want to use a dead letter queue (DLQ) to handle any messaging errors, create a Kafka topic to use for the DLQ.

Reviewer: Resolving this comment because it seems to have been implemented.

endif::[]
* You created a Data Generator source {connectors} instance.
* For the data sink example, open the free https://webhook.site[webhook.site^] in a browser window. The `webhook.site` page provides a unique URL that you copy for use as an HTTP data sink.

> * If you want to use a dead letter queue (DLQ) to handle any messaging errors, create a Kafka topic to use for the DLQ.

Reviewer: Maybe remove the "to use" here again, as you did in the data source section?
```suggestion
* If you want to use a dead letter queue (DLQ) to handle any messaging errors, create a Kafka topic for the DLQ.
```

> * If you want to use a dead letter queue (DLQ) to handle any messaging errors, create a Kafka topic to use for the DLQ.

Reviewer: Resolving as Done, thank you!

+
For example, select *test* and then click *Next*.

. On the *Namespace* page, the namespace that you select depends on your {openshift} environment.
+
If you're using your own {openshift} environment (ROSA or {osd-name-short} Trial), select the card for the namespace that was created when a cluster administrator added the {connectors} service to your cluster.
+
If you're using the hosted preview environment, click the *preview namespace* that you provisioned when you created the source connector.

> 

Reviewer: I can't tag the following lines to add my comments there directly:

1. Line 251: You might want to edit as follows:
- Original: "Select one of the following error handling policy for your ..."
- Proposed: "Select one of the following error handling polic**ies** for your ..."

2. Line 321: I can't check the UI, but just wanted to confirm that the field name is "Topic Name**s**" (plural) and not "Topic Name"?

3. Line 328: Move "Click **Next**." to a new sentence, because it's not part of selecting an error policy?
